### DISCUSSION TITLE
JavaScript SDK?

### DISCUSSION THREAD

**Discussion by TaToTanWeb** [deta/cloud-docs#388] [Help] [answered]
Hello,
the docs [here](https://docs.deta.sh/docs/base/sdk) for the deta base say that there are SDKs for JavaScript for both NodeJS and browser. I'm trying to access the API from browser, but the guide apparently has instructions for NodeJS only. Is there an SDK for browser too? And in case there's not, can I access the HTTP API from there without falling into CORS issues?
Thank youu

**Comment by jnsougata** [comment]
No, you will run into CORS issue

**Comment by lihugang** [comment, marked as answer]
I think you can use a micro as a proxy to solve cors problem.
